src/System.Management.Automation/engine/Interop/Windows/CreateIoCompletionPort.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static unsafe partial class Windows
+    {
+        internal sealed class SafeIoCompletionPort : SafeHandle
+        {
+            public SafeIoCompletionPort() : base(nint.Zero, true) { }
```

Comment:
```suggestion
            public SafeIoCompletionPort() : base(invalidHandleValue: nint.Zero, ownsHandle: true) { }
```

Reply:
Seems a bit overkill when we don't seem to do it anywhere else but ok.

Reply:
Just style for readability.